pages/team.vue

```diff
@@ -65,5 +59,31 @@ const { data: team } = await useAsyncData('team', () =>
     fetchLinks: ['accordionitem.title', 'accordionitem.slices']
   })
 )
-usePrismicSEO(team.value?.data)
+usePrismicSEO(team.value.data)
+const teamStructure = team.value.data.slices1.reduce((accumulator, member) => {
+  const department = member.primary.department
+  if (department !== 'Founders' && department !== 'Alumni') {
+    const existingTeamIndex = accumulator.findIndex(team => team.teamName === department)
+
+    if (existingTeamIndex >= 0) {
+      accumulator[existingTeamIndex].members.push(member)
+    } else {
+      accumulator.push({ teamName: department, members: [member] })
+    }
+  }
+  return accumulator
+}, []).sort((a, b) => a.teamName.localeCompare(b.teamName))
+const foundersTeam = team.value.data.slices1.reduce((accumulator, member) => {
```

Comment:
for this case it is not necessary to use a `reduce` function since you want exactly one department so it's not really necessary to this grouping.

A much simpler way is to use in this case the `filter` method as following:
`const foundersTeam = team?.value?.data.slices1.filter(team => team.primary.department === 'Founders')`

This requires changing some things in the `template` html code since the types changed but this works better